src/Utilities/ProjectData.h

```diff
@@ -82,8 +86,13 @@ class ProjectData : public OhmmsElementBase
 
   int getSeriesIndex() const { return m_series; }
   int getMaxCPUSeconds() const { return max_cpu_secs_; }
+  const std::string& get_driver_epoch() const { return driver_epoch_; }
+
+  const std::string& getName() const { return name_; }
+  void setName(const std::string& aname) { name_ = aname; }
 
 private:
+  std::string name_;
```

Comment:
What is this name? Isn't m_title enough?

Reply:
In removing pointless and obfuscating inheritance from OhmmsElementBase I preserved the only functionality from the base class it used, having a name member.

Reply:
I didn't notice the removal of the base class. Yes it was pointless.
I tried removing name_ getName setName. They don't affect qmcpack. So let us clean this up. Some unit test correction is needed.